homeassistant/components/braviatv/config_flow.py

```diff
@@ -129,9 +130,12 @@ async def async_step_authorize(self, user_input=None):
                 return self.async_create_entry(title=self.title, data=user_input)
 
         # Connecting with th PIN "0000" to start the pairing process on the TV.
-        await self.hass.async_add_executor_job(
-            self.braviarc.connect, "0000", CLIENTID_PREFIX, NICKNAME,
-        )
+        try:
+            await self.hass.async_add_executor_job(
+                self.braviarc.connect, "0000", CLIENTID_PREFIX, NICKNAME,
+            )
+        except JSONDecodeError:
```

Comment:
Shouldn't this error be caught by the library and another library specific error raised? How should the user of the library know that this error means that IP Control isn't supported?

Reply:
Yes, you're right. I'm closing this PR.